src/objects/GetStartedChecklistSection__mdt.object

```diff
@@ -1,6 +1,32 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <CustomObject xmlns="http://soap.sforce.com/2006/04/metadata">
     <description>NPSP Get Started Page sections</description>
+    <fields>
+        <fullName>Belongs_to__c</fullName>
+        <description>To select in which checklist has displayed this section</description>
+        <externalId>false</externalId>
+        <fieldManageability>DeveloperControlled</fieldManageability>
+        <inlineHelpText>To select in which checklist has displayed this section</inlineHelpText>
+        <label>Belongs to</label>
```

Comment:
```suggestion
        <fullName>Page_Type__c</fullName>
        <description>The type of Get Started page (Admin or End User) where these sections will appear.</description>
        <externalId>false</externalId>
        <fieldManageability>DeveloperControlled</fieldManageability>
        <inlineHelpText>The type of Get Started page (Admin or End User) where these sections will appear.</inlineHelpText>
        <label>Page Type</label>
```

Reply:
@fparodi-salesforce I just realized that we need to make one more small change. This field is on a single section, right? I think it should be "The type of Get Started page (Admin or End User) where this section will appear."